frontend/lib/api.js

```diff
@@ -55,3 +55,16 @@ export async function loadPodcasts() {
     });
   return res.items;
 }
+
+export async function loadPastEvents() {
+  const res = await client
+    .getEntries({
+      content_type: "pastEvents",
+      select: "fields",
+      order: "fields.index",
```

Comment:
The index should be reversed

Example: oweek recruitment should be the first event of T1 2022 (ie at the bottom of the past events page) and the cisco event is the last event of T1 2022